bazel/repository_locations.bzl

```diff
@@ -425,8 +425,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_url = "https://luajit.org",
         # The last release version, 2.1.0-beta3 has a number of CVEs filed
         # against it. These may not impact correct non-malicious Lua code, but for prudence we bump.
-        version = "1d8b747c161db457e032a023ebbff511f5de5ec2",
-        sha256 = "20a159c38a98ecdb6368e8d655343b6036622a29a1621da9dc303f7ed9bf37f3",
+        version = "a04480e311f93d3ceb2f92549cad3fffa38250ef",
```

Comment:
i was going to comment that we should use a tagged release as new versions would be flagged by our deps system, but then i read https://github.com/LuaJIT/LuaJIT/issues/563#issuecomment-1206294291 so looks like that wont happen

Reply:
Yes, it seems that Mike Pall is tired of making new release.